apps/labs/services/posts/serializePost.ts

```diff
@@ -33,23 +33,24 @@ export const serializePost = async (
     },
   });
 
-  if (!data.author) {
+  if (!data.author || data.author.length == 0) {
     throw Error('You did not provide author slug');
   }
 
-  const postAuthor = authors.find((author) => author.slug === data.author);
-
-  if (!postAuthor) {
-    throw Error('There is not matching author in database');
-  }
-
+  const postAuthor = data.author.map((authorName) => {
```

Comment:
I think this should be plural, `postAuthors`, since it can now represent more than one author?

It would be good to add typing to this, if possible, to make it clear (a) what the type is and (b) that it's an array